src/guides/v2.3/comp-mgr/patching.md

```diff
@@ -75,14 +79,164 @@ index c8a6fef58d31..7d01c195791e 100644
 
 ## Applying patches
 
-There are two ways to apply patches:
+There are three ways to apply patches:
```

Comment:
If you say three, should the list be numbered? I don't mind either, just wondering what the threshold is.

Reply:
I think that's subjective. I prefer using ordered lists when sequence is important.